src/main/scala/variant/Antichess.scala

```diff
@@ -38,6 +38,8 @@ case object Antichess
   override def valid(situation: Situation, strict: Boolean) =
     situation.board.nbPieces >= 2 && situation.board.nbPieces <= 32
 
+  override def hasValidCheckers(strict: Boolean, situation: Situation) = true
```

Comment:
If we removed `hasValidCheckers` from Variant, we don't need to do anything here.